tests/ui/README.md

```diff
@@ -0,0 +1,68 @@
+## CodaLab UI Tests
+
+We test the CodaLab UI by using [Selenium](https://pypi.org/project/selenium/#history) and 
+[diffimg](https://pypi.org/project/diffimg/#history) for our screenshots comparison tool. Currently, our front end 
+automation only runs in Chrome and Firefox. We hope to support more browsers in the future. 
+
+### Setting up the tests on your local machine
+
+#### One-time setup
+
+1. If you haven't already, install the Python dependencies specified in `requirements.txt`, located in the root of the 
+repository, by running `pip install -r requirements.txt`.
+2. Next, install [ChromeDriver](https://chromedriver.chromium.org/getting-started) (for Chrome) and 
+[GeckoDriver](https://github.com/mozilla/geckodriver) (for Firefox). These drivers are the links between the Selenium 
+tests and their respective browsers.
+    1. Run `./scripts/test-setup.sh` and place the installed GeckoDriver on your path by running 
+    `export PATH=$PATH:/usr/bin/geckodriver`. This is not needed for ChromeDriver.
+    2. An alternative is to simply use [Homebrew](https://brew.sh/) and run `brew install geckodriver && brew cask 
+    install chromedriver`. 
+
+#### Running the tests locally
+
+1. From the root of the repository, start the CodaLab service by running `./codalab_service.py start -bd`.
+2. Populate your local machine with a comprehensive sample worksheet by running 
+`docker exec -it codalab_rest-server_1 /bin/bash -c "python3 scripts/create_sample_worksheet.py"`. 
```

Comment:
Instead of this, can we just ask them to run the command that runs the `worksheets` module in `test_cli`? This way, the local setup and travis setup will be more uniform.

Reply:
If I want to run and target a specific test, do I run `docker exec -it codalab_rest-server_1 /bin/bash -c "python3 test_cli.py worksheets"`? This seems to be failing for me on my local machine.

Reply:
I think the current instruction on creating a sample worksheet is quick and should be good enough for now.